tests/readme_examples.cpp

```diff
@@ -305,6 +305,30 @@ bool is_correct_string() {
   return is_ok;
 }
 
+void parse_documentation() {
+  const char *json      = "{\"key\":\"value\"}";
+  const size_t json_len = strlen(json);
+  simdjson::dom::parser parser;
+  // We comment out element since it is unused.
+  //simdjson::dom::element element = 
```

Comment:
Could these two lines be removed?

Reply:
@pauldreik We try to match the code to our examples as closely as possible. In this instance, I could not because of spurious warnings. However, I can uncomment and add an extra line to avoid the unused warning.

Reply:
Done.